mobile/src/components/shared/ICardPage.jsx

```diff
@@ -64,24 +95,49 @@ const MyICardPage = (props) => {
             </View>
           </View>
         )}
+     
+      //Adding two new experimental status
+
+      //inactive, verify
+      else if (status == 'iNactive') {
+        content = (
+            <View style={styles.notification}>
+            <Image source={require('../../../assets/x.png')} style={styles.notificationPic} />
+            <View justifyContent={'center'}>
+              <Text style={styles.notificationText}>Your account is unverified. Please{'\n'}go through the verification process</Text>
+            </View>
+          </View> 
+        )}
+          //inactive, re
+        else if (status == 'Inactive') {
+          content = (
+              <View style={styles.notification}>
+              <Image source={require('../../../assets/x.png')} style={styles.notificationPic} />
+              <View justifyContent={'center'}>
+                <Text style={styles.notificationText}>Something went wrong, please{'\n'}reverify account or contact ISA at{'\n'}isa.general@ualberta.ca </Text>
+              </View>
+            </View> 
+          )}
+
     return <View>{content}</View>
   }
- 
   return (
     <ImageBackground source={require('../../../assets/Background.png')} resizeMode="cover" style={styles.backgroundImage}>
  
      
       <Card status={props.status} />
-     
- 
-      <View style={styles.container} backgroundColor={statusColors[props.status]} marginTop={props.status == 'inactive' || props.status == 'stale' || props.status == 'Unlinked' ? 109 : 235}>
+
+      <View style={styles.container} backgroundColor={statusColors[props.status]} marginTop={props.status == 'inactive' || props.status == 'stale' || props.status == 'Unlinked'||props.status =='iNactive' || props.status == 'Inactive' ? 109 : 235}>
         <View style={styles.containerInside}>
           <Image source={user != null ? {uri:user.picture } : require('../../../assets/account.png')} style={styles.avatar} borderColor={statusColors[props.status]} />
           <Text style={styles.userName} >{user != null ? user.name : 'N/A'}</Text>
           <Text style={styles.statusHeader}>ISAF status</Text>
           <Text style={[{...styles.ISAFStatus, color: statusColors[props.status]}]} >{props.status}</Text>
         </View>
       </View>
+
+      {/* Experimental Position */}
+      <Button onPress={props.verify} status={props.status}/>
```

Comment:
We should probably make this button conditionally render only if the user needs to verify.